rippled.cc

```diff
@@ -43,6 +43,7 @@ void sighandler(int signo) {
 }
 
 int main(int argc, char **argv) {
+  gflags::ParseCommandLineFlags(&argc, &argv, true);
```

Comment:
Can you put this into mysql_ripple::Init()? I think it will be more appropriate to keep all initialization there.

Reply:
Yes I agree